apps/scan/frontend/src/components/display_settings_button.tsx

```diff
@@ -0,0 +1,34 @@
+/* stylelint-disable order/properties-order */
+import React from 'react';
+import { Button, Icons } from '@votingworks/ui';
+import styled from 'styled-components';
+import { useHistory } from 'react-router-dom';
+import { Paths } from '../constants';
+
+const LabelContainer = styled.span`
+  align-items: center;
+  display: flex;
+  flex-wrap: nowrap;
+  font-weight: ${(p) => p.theme.sizes.fontWeight.semiBold};
+  gap: 0.5rem;
+  text-align: left;
+  width: min-content;
```

Comment:
Curious what this would look like with the button text all on one line. Maybe it would seem less intrusive on the screen?

Reply:
Yeah, I had it that way previously and it looked odd at the larger sizes (mostly weird wrapping at the largest size) - but thought about it a bit over the weekend and want to try again with some workarounds (esp. considering people aren't going to be flipping back and forth between sizes)
Will add a screen grab of the the single line option for comparison and try out Roe's copy suggestions as well